services/QuillLMS/client/app/bundles/Evidence/actions/session.ts

```diff
@@ -123,49 +118,44 @@ export const saveActiveActivitySession = ({ completedSteps, timeTracking, studen
           studentHighlights,
         }
       },
-      json: true,
-    }
-
-    request.put(requestObject, (e, r, body) => {
-      if (callback) callback()
-    })
+      (body) => {
+        if (callback) callback()
+      }
+    )
   }
 }
 
 export const saveActivitySurveyResponse = ({ sessionID, activitySurveyResponse, callback, }) => {
-  const activeActivitySessionUrl = `${process.env.DEFAULT_URL}/api/v1/activity_survey_responses`
-
-  const requestObject = {
-    url: activeActivitySessionUrl,
-    body: {
+  requestPost(
+    `${process.env.DEFAULT_URL}/api/v1/activity_survey_responses`,
+    {
       activity_survey_response: activitySurveyResponse,
       activity_session_uid: sessionID
     },
-    json: true,
-  }
-
-  request.post(requestObject, (e, r, body) => {
-    if (callback) callback()
-  })
+    (body) => {
+      if (callback) callback()
+    },
+    (body) => {
```

Comment:
The simplest explanation is that originally there wasn't, and then in testing this request failed on anonymous activity sessions and the user couldn't progress, so I fixed it. So it was just the only place that I noticed that there needed to be a matching error block too. That should have triggered me to go look for where else this pattern existed - I'll do that now before this gets released in case there's a hidden error somewhere.